server/service/src/localisations.rs

```diff
@@ -0,0 +1,87 @@
+use std::{collections::HashMap, fs };
+use serde_yaml::Value;
+
+use rust_embed::RustEmbed;
+
+#[derive(RustEmbed)]
+// Relative to server/Cargo.toml
+// later this will be client in dev mode, or build in production mode
+#[folder = "../../client/packages/host/dist"]
+pub struct EmbeddedLocalisations;
+
+
+
+// struct to manage translations
+pub struct Localisations {
+    pub translations: HashMap<String, HashMap<String, String>>,
+}
+
+pub struct TranslationStrings {
+    pub translations: HashMap<String, Value>,
+}
+
+impl Localisations {
+
+    // Creates a new Localisations struct
+    pub fn new() -> Self {
+        Localisations {
+            translations: HashMap::new(),
+        }
+    }
+
+    // Load translations from embedded files
+    pub fn load_translations(&mut self) -> Result<(), std::io::Error> {
+        // Languages - need to extract these from the files themselves?
+        let mut languages = Vec::new();
+
+        // read all dirs within locales dir
+        let folder_paths: Vec<String> = fs::read_dir("../client/packages/host/dist/locales")
```

Comment:
This is not available in production, I think you would use `EmbeddedLocalisations::iter()`